pkg/clients/http.go

```diff
@@ -4,7 +4,6 @@ import (
 	"net/http"
 )
 
-//go:generate mockgen -source=http.go -package=clients -destination=./http_mock.go
```

Comment:
For some reason it did these to several. It feels like it started running the `mockgen` command itself- and deleted the lines which let it auto generate using `make generate`

Reply:
Same - now in Makefile.